msal/src/test/java/com/microsoft/identity/client/robolectric/tests/mocked/CommandResultCachingTest.java

```diff
@@ -0,0 +1,346 @@
+                        .forceRefresh(false)
+                        .fromAuthority(AAD_MOCK_AUTHORITY)
+                        .withClaims(cr)
+                        .withCallback(new CacheCountAuthenticationCallback(250))
+                        .build();
+
+                RoboTestUtils.flushScheduler();
+
+                publicClientApplication.acquireTokenSilentAsync(silentParameters);
+
+                RoboTestUtils.flushScheduler();
+            }
+
+        }.instantiatePCAthenAcquireToken();
+    }
+
+
+
+    private IAccount performGetAccount(IPublicClientApplication application, final String loginHint) {
+        final IAccount[] requestedAccount = {null};
+        final IMultipleAccountPublicClientApplication multipleAcctApp = (IMultipleAccountPublicClientApplication) application;
+        multipleAcctApp.getAccount(
+                loginHint.trim(),
+                new IMultipleAccountPublicClientApplication.GetAccountCallback() {
+                    @Override
+                    public void onTaskCompleted(final IAccount account) {
+                        if (account != null) {
+                            requestedAccount[0] = account;
+                        } else {
+                            fail("No account found matching identifier");
+                        }
+                    }
+
+                    @Override
+                    public void onError(final MsalException exception) {
+                        fail("No account found matching identifier");
+                    }
+                });
+        RoboTestUtils.flushScheduler();
+        return requestedAccount[0];
+    }
+
+    private ICacheRecord createDataInCache(IPublicClientApplication application) {
+        ICacheRecord cacheRecord = null;
+        final TokenResponse tokenResponse = MockTokenResponse.getMockSuccessTokenResponse();
+
+        try {
+            cacheRecord = RoboTestCacheHelper.saveTokens(tokenResponse, application);
+        } catch (ClientException e) {
+            fail("Unable to save tokens to cache: " + e.getMessage());
+        }
+
+        return cacheRecord;
+    }
+
+    private ICacheRecord createDataInCacheWithExpiredAccessToken(IPublicClientApplication application) {
+        ICacheRecord cacheRecord = null;
+        final TokenResponse tokenResponse = MockTokenResponse.getMockTokenResponseWithExpiredAccessToken();
+
+        try {
+            cacheRecord = RoboTestCacheHelper.saveTokens(tokenResponse, application);
+        } catch (ClientException e) {
+            fail("Unable to save tokens to cache: " + e.getMessage());
+        }
+
+        return cacheRecord;
+    }
+
+    private IAccount loadAccountForTest(IPublicClientApplication application) {
+        ICacheRecord cacheRecord = createDataInCache(application);
+        final String loginHint = cacheRecord.getAccount().getUsername();
+        final IAccount account = performGetAccount(application, loginHint);
+        return account;
+    }
```

Comment:
Are these methods needed? They are not being used anywhere.

Reply:
Yep.  Leftover from when creating the tests.  Deleted.